src/Polysemy/ConstraintAbsorber/MonadCont.hs

```diff
@@ -28,7 +28,7 @@ absorbCont
        -- 'Sem'. This might be something with type @'C.MonadCont' m => m a@.
     -> Sem r a
 absorbCont = absorbWithSem @C.MonadCont @Action
-  (ContDict (callCC @ref))
+  (ContDict (\main -> callCC @ref $ \exit -> main exit))
```

Comment:
Out of curiosity, does simply `(ContDict (\main -> callCC @ref main))` work? My guess is no.

Reply:
nope, can't match `a1 -> Sem r b` with `forall b1. a1 -> Sem r b1`.

Reply:
Suspected as much.